tkltest/generate/unit/generate.py

```diff
@@ -178,9 +178,17 @@ def generate_ctd_amplified_tests(config, output_dir):
                      num_executions=config['generate']['ctd_amplified']['num_seq_executions'],
                      test_directory=tmp_test_directory, verbose=verbose)
 
-    if os.path.exists(test_directory):
-        shutil.rmtree(test_directory)
-    shutil.move(tmp_test_directory, test_directory)
+    if not os.path.isdir(os.path.join(tmp_test_directory, 'monolithic')):
```

Comment:
@haim-kermany we should support an empty test suite instead of exiting, because we run augmentation on it and can eventually get a test suite (remember we had an issue on this)